tutorials/containers/using-gitlab-as-a-container-registry/using-gitlab-as-a-container-registry.md

```diff
@@ -0,0 +1,244 @@
+---
+id: using-gitlab-as-a-container-registry
+summary: Learn how to use GitLab as a private container registry
+categories: containers
+tags: kubernetes, containers, gitlab, cicd
+difficulty: 3
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-09-02
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+---
+
+# Using GitLab as a container registry
+
+## Introduction
+
+The aim of this tutorial post is to demonstrate how you can use Charmed Kubernetes in a CI/CD environment using GitLab’s container registry functionality.
+
+GitLab has the ability to store upto 10GB in a container registry for projects, you can incorporate the building of these containers into your own CI/CD pipeline or you can use Gitlab’s own CI/CD functionality to do this for you. For the purposes of this blog post you will do this by hand so you can get a grasp of the process.
+
+## Before you start
```

Comment:
Make it a sub-section of the "Overview" section ("You will only need ..."). Again, you can reference some of the latest tutorials.